<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AI助手</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-image: url(:/ui/images/background.png);
    background-position: center;
    background-repeat: no-repeat;
    background-attachment: fixed;
    background-color: #f0f0f0;
}
QTextEdit, QPlainTextEdit {
    background-color: rgba(255, 255, 255, 220);
    border: 1px solid #999;
    border-radius: 4px;
    padding: 4px;
}
QLabel {
    color: #333333;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>输入问题：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditUserQuery">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>文件内容</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>文件1：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditFilePath1">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>选择的文件路径将显示在这里</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_selectFile">
           <property name="text">
            <string>选择文件</string>
           </property>
           <property name="icon">
            <iconset theme="document-open">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditFileContent">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="placeholderText">
          <string>文件1的内容将显示在这里</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>文件2：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditFilePath2">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>选择的文件路径将显示在这里</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_selectFile2">
           <property name="text">
            <string>选择文件</string>
           </property>
           <property name="icon">
            <iconset theme="document-open">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditFileContent2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="placeholderText">
          <string>文件2的内容将显示在这里</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>AI回答</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>2</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>AI的回答将显示在这里</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="AiServerConfig">
        <property name="text">
         <string>AI服务器配置</string>
        </property>
        <property name="icon">
         <iconset theme="preferences-system">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>设置AI助手角色</string>
        </property>
        <property name="icon">
         <iconset theme="preferences-desktop-user">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>语音设置</string>
        </property>
        <property name="icon">
         <iconset theme="audio-volume-high">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>询问AI</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>aiassist.png</normaloff>aiassist.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>系统</string>
    </property>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menuview">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="actionchathistory"/>
    <addaction name="actioncommonprompt"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>知识库</string>
    </property>
    <addaction name="actionfileimport"/>
   </widget>
   <addaction name="menuSystem"/>
   <addaction name="menuview"/>
   <addaction name="menu"/>
  </widget>
  <action name="actionexit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionchathistory">
   <property name="text">
    <string>聊天历史</string>
   </property>
  </action>
  <action name="actioncommonprompt">
   <property name="text">
    <string>常用系统提示</string>
   </property>
  </action>
  <action name="actionfileimport">
   <property name="text">
    <string>文件导入</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
